doc/reference_kinds.md

```diff
@@ -13,18 +13,18 @@ extra field to the `Location` type:
 
 ```typescript
 
-export type AlsReferenceKind = 'read' | 'write' | 'call';
+export type AlsReferenceKind = 'w' | 'c' | 'd';
 
 export namespace AlsReferenceKind {
-   export const Read  : AlsReferenceKind = 'read';
-   export const Write : AlsReferenceKind = 'write';
-   export const Call  : AlsReferenceKind = 'call';
+   export const Write            : AlsReferenceKind = 'w';
```

Comment:
to prepare for the future, maybe we could add support for "Label", and for "Read" (in Ada we can have both Read+Write for an object that's passed as "in out")

Reply:
I don't see why we should do it in advance. For me it's better to keep implementation in sync with specification. Besides 'Label' and 'Read' we already have a lot of (useless) kinds, I doubt anyone uses it ever.